portal/i18next_gulpfile.js

```diff
@@ -50,21 +50,25 @@ function save(target) {
  */
 gulp.task('i18next-extraction', ['clean-src'], function() {
     console.log('extracting text and generate json file ...');
-    return gulp.src(['static/**/*.{js,html}'])
+    return gulp.src(['static/**/*.{js,html}', 'templates/*.html', 'gil/templates/gil/*.html', 'eproms/templates/eproms/*.html'])
```

Comment:
I just realized we'll need to separate ePROMs and TrueNTH content when uploading to Smartling- can you either add an option for specifying eproms/gil, or just split the extract command into two commands?